src/test/kotlin/com/chloe/kotlinserv/TestGetCountryStats.kt

```diff
@@ -0,0 +1,288 @@
+package com.chloe.kotlinserv
```

Comment:
please follow this template all tests

1. every mock, every variable init in first block (no empty lines)
2. empty line
3. test method
4. empty line
5. all verifies and assertions related stuff